From: Jan Engelhardt <jengelh@inai.de>
Date: 2020-08-29 10:10:12.022296526 +0200

Avoid the installation of a custom xxhash.h header. Since vk_*.h header files
still include xxhash.h (and now would source the system xxhash instead of the
bundled one), it makes sense to build ValidationLayers itself with the system
xxhash as well, so that the ABI matches.

---
 layers/CMakeLists.txt |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

Index: Vulkan-ValidationLayers-sdk-1.3.250.0/layers/CMakeLists.txt
===================================================================
--- Vulkan-ValidationLayers-sdk-1.3.250.0.orig/layers/CMakeLists.txt
+++ Vulkan-ValidationLayers-sdk-1.3.250.0/layers/CMakeLists.txt
@@ -26,8 +26,6 @@ target_sources(VkLayer_utils PRIVATE
     containers/custom_containers.h
     error_message/logging.h
     error_message/logging.cpp
-    external/xxhash.h
-    external/xxhash.cpp
     ${API_TYPE}/generated/lvt_function_pointers.cpp
     ${API_TYPE}/generated/lvt_function_pointers.h
     ${API_TYPE}/generated/vk_format_utils.h
@@ -66,7 +64,7 @@ target_sources(VkLayer_utils PRIVATE
 # https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/4640
 target_compile_definitions(VkLayer_utils PUBLIC XXH_NO_LONG_LONG)
 
-target_link_libraries(VkLayer_utils PUBLIC Vulkan::Headers)
+target_link_libraries(VkLayer_utils PUBLIC Vulkan::Headers -lxxhash)
 target_include_directories(VkLayer_utils SYSTEM PRIVATE external)
 target_include_directories(VkLayer_utils PUBLIC . ${API_TYPE})
 
